doc/sources/samples.rst

```diff
@@ -53,6 +53,14 @@ If you want to run them locally, refer to `these instructions
    Kmeans for spoken arabic digit dataset <samples/kmeans.ipynb>
    DBSCAN for spoken arabic digit dataset <samples/dbscan.ipynb>
 
+.. rubric:: Jupyter Notebook Samples
```

Comment:
No please-  those notebooks are not validated and not monitored. I wouldn't like to push people there. 

There are notebooks that are available for this within the repo and i would push to those.

Reply:
@napetrov  These JN samples are monitored and validated by the oneAPI samples team, and used in promotion and also included in Intel DevCloud by default - these need to be kept.

Reply:
This is probably separate question but realistically they have to be improved/replaced - "Intel Python sklearn Getting Started Example for Shared Memory Systems"

Normal AI people are not talking in terms of "Shared Memory Systems" - this is HPC term
Intel Python sklearn - this even not a thing, even not talking that it's incorrect use of the names. 

Second notebook stored without outputs which at least looking strange there

And if those are great and supported already by marketing they probably don't need more attention from docs.  I'm fine with kipping links but as you see those samples kind of in strange shape

Reply:
Sorry what is the JN in question? The samples are just getting started and features and functionality
   Intel® Extension for Scikit-Learn* Getting Started Sample <https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_SKLearn_GettingStarted>
   Intel® Extension for Scikit-Learn*: SVC for Adult dataset Performance Sample <https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Features-and-Functionality/Intel_Extension_For_SKLearn_Performance_SVC_Adult>